subsys/bluetooth/services/ots/ots_client.c

```diff
@@ -1263,6 +1287,49 @@ static int oacp_write(struct bt_conn *conn, struct bt_otc_internal_instance_t *i
 
 	return err;
 }
+
+static int oacp_checksum(struct bt_conn *conn, struct bt_otc_internal_instance_t *inst,
+			 uint32_t offset, uint32_t len)
+{
+	int err;
+
+	LOG_DBG("oacp_checksum");
+
+	if (!inst->otc_inst->oacp_handle) {
+		LOG_DBG("Handle not set");
+		return -EINVAL;
+	} else if (inst->busy) {
+		return -EBUSY;
+	} else if (cur_inst) {
```

Comment:
you could similarly add debug prints to differentiate those two `-EBUSY` retcodes.

Reply:
Add two LOG_DBG to tell which busy statement is.